docs/api/en/helpers/PolarGridHelper.html

```diff
@@ -33,17 +33,17 @@ <h2>Examples</h2>
 
 		<h2>Constructor</h2>
 
-		<h3>[name]( [param:Number radius], [param:Number radials], [param:Number circles], [param:Number divisions], [param:Color color1], [param:Color color2] )</h3>
+		<h3>[name]( [param:Number radius], [param:Number sectors], [param:Number rings], [param:Number divisions], [param:Color color1], [param:Color color2] )</h3>
 		<p>
 		radius -- The radius of the polar grid. This can be any positive number. Default is 10.<br />
-		radials -- The number of radial lines. This can be any positive integer. Default is 16.<br />
-		circles -- The number of circles. This can be any positive integer. Default is 8.<br />
+		sectors -- The number of sectors the grid will be divided into. This can be any positive integer. Default is 16.<br />
+		rings -- The number of rings. This can be any positive integer. Default is 8.<br />
 		divisions -- The number of line segments used for each circle. This can be any positive integer that is 3 or greater. Default is 64.<br />
 		color1 -- The first color used for grid elements. This can be a [page:Color], a hexadecimal value and an CSS-Color name. Default is 0x444444 <br />
 		color2 -- The second color used for grid elements. This can be a [page:Color], a hexadecimal value and an CSS-Color name. Default is 0x888888
 		</p>
 		<p>
-		Creates a new [name] of radius 'radius' with 'radials' number of radials and 'circles' number of circles, where each circle is smoothed into 'divisions' number of line segments. Colors are optional.
+		Creates a new [name] of radius 'radius' with 'sectors' number of sectors and 'rings' number of rings, where each circle is smoothed into 'divisions' number of line segments. Colors are optional.
```

Comment:
Do you mind also updating the Chinese version of the doc page? 

https://threejs.org/docs/index.html#api/zh/helpers/PolarGridHelper

Just copy over the new English text so it can be translated again.

Reply:
Since this PR is closed, I opened a new one here: https://github.com/mrdoob/three.js/pull/24533